docs/system_requirements/using_podman.md

```diff
@@ -3,7 +3,7 @@
 _Testcontainers for Go_ supports the use of Podman (rootless or rootful) instead of Docker.
 
 In most scenarios no special setup is required in _Testcontainers for Go_.
-_Testcontainers for Go_ will automatically discover the socket based on the `DOCKER_HOST` or the `TC_HOST` environment variables.
+_Testcontainers for Go_ will automatically discover the socket based on the `DOCKER_HOST` environment variables.
```

Comment:
I intentionally remove the mention on `TC_HOST` did not add `TESTCONTAINERS_HOST_OVERRIDE` here because, from what I could tell by going through the source code for socket detection, I believe `TC_HOST` (or `TESTCONTAINERS_HOST_OVERRIDE`) environment variable was not used. `tc.host` in `.testcontainers.properties` seems to be used but not `TC_HOST` (or `TESTCONTAINERS_HOST_OVERRIDE`) environment variable.